packages/web/src/pages/index.tsx

```diff
@@ -60,104 +57,23 @@ const HomePage: React.FC<IndexPageProps> = ({ vars }) => {
               {currentUserLoading || (
                 <>
                   <Link href={`/user/${user?.username}`}>
-                    <a className="flex items-center space-x-4">
-                      <img
```

Comment:
Opps, I forgot to remove this part when doing Pull request since UserSection components has this already.